docs/src/tutorial.jl

```diff
@@ -71,23 +71,25 @@ probs = probabilities(o, x)
 
 maximum(probs)
 
-# Or iterate over the contained probabilities values
-s = 0.0
-for p in probs
-    s += p
-end
```

Comment:
Why has this been removed from the tutorial? It is important to show that `probs` can be used like any other array.

Reply:
This caused the docs to error after migrating to v1. Something about `s` not being defined. I don't understand why and could not fix it, so it was removed.

Reply:
To be clear: the code runs just fine, just not as part of a Literate.jl generated doc page.

Reply:
okay I know what is the problem I will fix it in my tutorial PR.